### DISCUSSION TITLE
Docs

### DISCUSSION THREAD

**Discussion by alphaolomi** [opcodesio/log-viewer#17] [Ideas]
Hello there, 

Great package, congrats @arukompas and contributors.

Proposing adding more documentations. the https://github.com/opcodesio/log-viewer/blob/main/README.md is a great start, Adding more sections will be nice (any plans to dedicated docs/wiki).

There is a handful source code facilitating log viewing, example facade and more. having some internal code comments indicating internal APIs and public APIs.

Would be nice adding docs to help building custom logs viewing methods.

Thanks.

**Comment by arukompas** [comment]
I am planning on making a dedicated website (like I have one for [Spike documentation](https://spike.opcodes.io/docs)) for the Log Viewer as well in the future.

**Comment by alphaolomi** [comment]
That will be great!! And again thank you.
